### PR TITLE
Fix/plugin build

### PR DESCRIPTION
**Pull request by ChrisCindy**
- [x] plugin-docusaurus: update docusaurus version and remove react/react-dom from devDependencies
- [x] plugin-docusaurus: use _createElement as praga which is exported by hijackCreateElement to support rpx
- [x] plugin-rax-component: external helpers
- [x] pkg: add install @swc/helpers warning if needed 
- [x] pkg: update externals default value from true to false and update doc
- [x] pkg: remove rpx built-in support (which will be implemented by plugins) 
- [x] pkg: fix check chunk.code in reportSize in build task
- [x] pkg: use classic runtime to compile JSX in bundle task

### CODE REVIEW

File: packages/plugin-docusaurus/src/configureDocusaurus.mts [hunk 1]
```diff
@@ -58,23 +58,63 @@ plugins: [
...
+        'rax-is-valid-element': require.resolve('rax-compat/is-valid-element'),
+        'rax-unmount-component-at-node': require.resolve('rax-compat/unmount-component-at-node'),
       }
     }
   ]
 ]
 `;
+
+  const hijackCreateElementModuleContent = `
+  import { createElement } from 'react';
```
Comment (wssgcg1213) [inline packages/plugin-docusaurus/src/configureDocusaurus.mts:77]: `import { createElement as _createElement } from 'react';`

File: packages/plugin-docusaurus/src/configureDocusaurus.mts [hunk 2]
```diff
@@ -58,23 +58,63 @@ plugins: [
...
+          convertedStyle[prop] = convertUnit(style[prop]);
+        }
+        result['style'] = convertedStyle;
+        return result;
+      }
+    }
+    return props;
+  }
+
+  export default function _createElement(component, props, ...children) {
```
Comment (wssgcg1213) [inline packages/plugin-docusaurus/src/configureDocusaurus.mts:101]: 导出名是 `createElement` 合适一些

File: packages/plugin-docusaurus/src/configureDocusaurus.mts [hunk 3]
```diff
@@ -58,23 +58,63 @@ plugins: [
...
+
   const babelConfigContents = `
 module.exports = {
-  presets: [require.resolve('@docusaurus/core/lib/babel/preset')],
+  presets: [
+    require.resolve('@docusaurus/core/lib/babel/preset'),
+    [
+      require.resolve('@babel/preset-react'),
+      {
+        "pragma": "_createElement",
```
Comment (wssgcg1213) [inline packages/plugin-docusaurus/src/configureDocusaurus.mts:113]: 这里也同步~

File: website/docs/scenarios/react.md [hunk 4]
```diff
@@ -97,11 +97,18 @@ import { defineConfig } from '@ice/pkg';
 export default defineConfig({
   bundle: {
     formats: ['esm', 'es2017'],
-    externals: true,
+    externals: {
+      react: 'React',
+      'react-dom': 'ReactDOM'
```
Comment (wssgcg1213) [inline website/docs/scenarios/react.md:102]: `,`

File: website/docs/scenarios/react.md [hunk 5]
```diff
@@ -97,11 +97,18 @@ import { defineConfig } from '@ice/pkg';
...
+    }
```
Comment (wssgcg1213) [inline website/docs/scenarios/react.md:103]: `,`

**Comment by wssgcg1213** [comment]
image
@ChrisCindy

**Comment by wssgcg1213** [comment]
可以 check 下为啥站点构建挂了, 一般来说是 md 文件引用路径错了, website 目录本地用 yarn 安装依赖

**Review by wssgcg1213 — APPROVED** [review]

**Review by wssgcg1213 — APPROVED** [review]

**Review by ClarkXia — APPROVED** [review]

File: packages/plugin-docusaurus/src/configureDocusaurus.mts [hunk 6]
```diff
@@ -58,23 +58,63 @@ plugins: [
...
+        'rax-unmount-component-at-node': require.resolve('rax-compat/unmount-component-at-node'),
       }
     }
   ]
 ]
 `;
+
+  const hijackCreateElementModuleContent = `
+  import { createElement as _createElement } from 'react';
+  import { convertUnit } from 'style-unit';
```
Comment (maoxiaoke) [inline packages/plugin-docusaurus/src/configureDocusaurus.mts:78]: 这里引入了一个 style-unit 的 package，但是没有依赖

**Pull request merged**